optics-core/src/Optics/Operators/Unsafe.hs

```diff
@@ -0,0 +1,37 @@
+-- |
+-- Module: Optics.Operators.Unsafe
+-- Description: Definitions of unsafe infix operators for optics.
+--
+module Optics.Operators.Unsafe
+  ( (^?!)
+  )
+  where
+
+import Data.Maybe
+import GHC.Stack
```

Comment:
We should explicit import lists for all external imports.

Reply:
We generally don't do this in other modules though.

Reply:
We should there as well.